src/app/components/game/game.component.ts

```diff
@@ -126,28 +134,31 @@ export class GameComponent implements OnInit, OnChanges {
         this.CURSOR_DATA.mouseIsDown = true;
         if (this.CURSOR_DATA.overSquare) {
             this.CURSOR_DATA.mouseDownOn = this.CURSOR_DATA.overSquare;
-            if (!this.isPromoting) {
-                this.CURSOR_DATA.dragging = true;
-            }
-            if (this.twoClickMove.attempting) {
-                this.CURSOR_DATA.mouseUpOn = this.CURSOR_DATA.mouseDownOn;
-                this.CURSOR_DATA.mouseDownOn = this.twoClickMove.source;
-                this.attemptMoveOnBoard();
-                this.twoClickMove.attempting = false;
-                this.twoClickMove.source = null;
-                this.CURSOR_DATA.mouseDownOn = this.CURSOR_DATA.overSquare;
-                this.tintSqFromMouseObjects = [];
-                this.drawBoard();
-                this.showMoves();
-            } else {
-                this.twoClickMove.attempting = false;
-                this.twoClickMove.source = null;
+            if (e.which == 1) {
```

Comment:
I would like some clarity about `e.which`, either in the form of a comment or by setting it to (and this is just a guess but I assume it is left click)

`let leftMouseClick = e.which === 1;`

(`true` if 1, `false` if anything else)

and then `if (leftMouseClick) { ... }`

Reply:
I'll write clarifying comments -- easier to understand

Reply:
Great, thanks.